hail/scripts/deploy.sh

```diff
@@ -80,6 +83,8 @@ curl -XPOST -H @$GITHUB_OAUTH_HEADER_FILE https://api.github.com/repos/hail-is/h
 
 retry skopeo copy $HAIL_GENETICS_HAIL_IMAGE docker://docker.io/hailgenetics/hail:$HAIL_PIP_VERSION
 retry skopeo copy $HAIL_GENETICS_HAIL_IMAGE docker://us-docker.pkg.dev/hail-vdc/hail/hailgenetics/hail:$HAIL_PIP_VERSION
+retry skopeo copy $HAIL_GENETICS_HAILTOP_IMAGE docker://docker.io/hailgenetics/hailtop:$HAIL_PIP_VERSION
+retry skopeo copy $HAIL_GENETICS_HAILTOP_IMAGE docker://us-docker.pkg.dev/hail-vdc/hail/hailgenetics/hailtop:$HAIL_PIP_VERSION
```

Comment:
I am suddenly wondering: how do we get these images into Azure?

Reply:
We don't. I want to run the `mirror_images.sh` script that I made recently as part of the CI process, maybe we could do that as part of #12378? I got stuck on the fact that we don't have a great way to synchronize when a release happens on GCP CI and azure CI grabbing the images from DockerHub, but every deploy should at least be much better than nothing.

Reply:
Or Azure could have a release process which also generates these common files? Why not do that?

Reply:
ah, duh, that is a good point

Reply:
For some reason I always think of it as strictly mirroring dockerhub and we only let one CI create the images that go to dockerhub..